.travis.yml

```diff
@@ -8,6 +8,7 @@ services:
 
 env:
   - TOXENV=django22
+  - TOXENV=django30
```

Comment:
shouldn't we also add a worker for Django 3.1

Reply:
Sure, I'll add that one too.